core/modules/admin_bar/js/admin_bar.js

```diff
@@ -1,547 +1,543 @@
-      Backdrop.adminBar.getCache(settings.admin_bar.hash, function (response) {
-        if (typeof response == 'string' && response.length > 0) {
-          $adminBar = $(response);
-          // Temporarily hide the admin bar while adjustments are made.
-          $adminBar.css({visibility: 'hidden'}).appendTo('body', context);
-          // Apply our behaviors.
-          Backdrop.adminBar.attachBehaviors(context, settings, $adminBar);
-          // Display the admin bar as soon as everything is done.
-          setTimeout(function () {
-            $adminBar.css({visibility: ''});
-          }, 0);
+    Backdrop.adminBar = Backdrop.adminBar || {};
+    Backdrop.adminBar.behaviors = Backdrop.adminBar.behaviors || {};
+    Backdrop.adminBar.cache = Backdrop.adminBar.cache || {};
+
+    /**
+     * Core behavior for Administration bar.
+     *
+     * Test whether there is an administration bar is in the output and execute all
+     * registered behaviors.
+     */
+    Backdrop.behaviors.adminBar = {
+        attach: function(context, settings) {
+            // Initialize settings.
+            settings.admin_bar = $.extend({
+                suppress: false,
+                margin_top: false,
+                position_fixed: false,
+                destination: '',
+                basePath: settings.basePath,
+                hash: 0,
+                replacements: {}
+            }, settings.admin_bar || {});
+            // Check whether administration bar should be suppressed.
+            if (settings.admin_bar.suppress) {
+                return;
+            }
+            var $adminBar = $('#admin-bar');
+            // Client-side caching; if administration bar is not in the output, it is
+            // fetched from the server and cached in the browser.
+            if (!$adminBar.length && settings.admin_bar.hash) {
+                Backdrop.adminBar.getCache(settings.admin_bar.hash, function(response) {
+                    if (typeof response == 'string' && response.length > 0) {
+                        $adminBar = $(response);
+                        // Temporarily hide the admin bar while adjustments are made.
+                        $adminBar.css({ visibility: 'hidden' }).appendTo('body', context);
+                        // Apply our behaviors.
+                        Backdrop.adminBar.attachBehaviors(context, settings, $adminBar);
+                        // Display the admin bar as soon as everything is done.
+                        setTimeout(function() {
+                            $adminBar.css({ visibility: '' });
+                        }, 0);
+                    }
+                });
+            }
+            // If the menu is in the output already, this means there is a new version.
+            else if (!$adminBar.hasClass('admin-bar-processed')) {
+                // Apply our behaviors.
+                Backdrop.adminBar.attachBehaviors(context, settings, $adminBar);
+            }
```

Comment:
Another coding standard problem: tabs should be only 2 spaces (you are using 4).

Reply:
My vscode has a mind of its own when it comes to this.